engine/apps/api/serializers/on_call_shifts.py

```diff
@@ -29,7 +29,7 @@ class OnCallShiftSerializer(EagerLoadingMixin, serializers.ModelSerializer):
         allow_null=True,
         required=False,
         child=UsersFilteredByOrganizationField(
-            queryset=User.objects, required=False, allow_null=True
+            queryset=User.objects, db_verification=True, required=False, allow_null=True
```

Comment:
I would suggest a different name for the param, based on my understanding of the check (see also the comment below), eg. `require_all_exist`?